src/axolotl/train.py

```diff
@@ -291,22 +314,158 @@ def terminate_handler(_, __, model_weakref):
         # defensively push to the hub to ensure the model card is updated
         trainer.push_to_hub()
 
-    return model, tokenizer
 
+def save_initial_configs(
+    cfg: DictDefault,
+    tokenizer: PreTrainedTokenizer,
+    model: PreTrainedModel,
+    peft_config: PeftConfig | None,
+):
+    """
+    Save initial configurations before training.
 
-def pretrain_hooks(_cfg, _trainer):
```

Comment:
Do we need to keep this? I saw this hook for a very long time, but I don't recall ever us using it @winglian .

Reply:
Oh yeah, I actually removed this in my telemetry PR. Unless @winglian has any reason to keep

Reply:
It's not used anywhere AFAICT so I think safe to remove.